documentation/package-table.md

```diff
@@ -25,236 +25,236 @@ Reference notes:
+[linux-musl-arm-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-musl-arm.tar.gz.sha512
 
 [linux-musl-arm64-badge-main]: https://aka.ms/dotnet/10.0.1xx/daily/linux_musl_arm64_Release_version_badge.svg?no-cache
 [linux-musl-arm64-version-main]: https://aka.ms/dotnet/10.0.1xx/daily/productCommit-linux-musl-arm64.txt
 [linux-musl-arm64-targz-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-linux-musl-arm64.tar.gz
-[linux-musl-arm64-targz-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-linux-musl-arm64.tar.gz.sha
+[linux-musl-arm64-targz-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-linux-musl-arm64.tar.gz.sha512
 
 [linux-musl-arm64-badge-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/linux_musl_arm64_Release_version_badge.svg?no-cache
 [linux-musl-arm64-version-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/productCommit-linux-musl-arm64.txt
 [linux-musl-arm64-targz-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-linux-musl-arm64.tar.gz
-[linux-musl-arm64-targz-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-linux-musl-arm64.tar.gz.sha
+[linux-musl-arm64-targz-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-linux-musl-arm64.tar.gz.sha512
 
 [linux-musl-arm64-badge-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/linux_musl_arm64_Release_version_badge.svg?no-cache
 [linux-musl-arm64-version-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/productCommit-linux-musl-arm64.txt
 [linux-musl-arm64-targz-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-musl-arm64.tar.gz
-[linux-musl-arm64-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-musl-arm64.tar.gz.sha
+[linux-musl-arm64-targz-checksum-9.0.3XX]: https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-musl-arm64.tar.gz.sha512
 
 [win-arm-badge-main]: https://aka.ms/dotnet/10.0.1xx/daily/win_arm_Release_version_badge.svg?no-cache
 [win-arm-version-main]: https://aka.ms/dotnet/10.0.1xx/daily/productCommit-win-arm.txt
 [win-arm-zip-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-win-arm.zip
-[win-arm-zip-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-win-arm.zip.sha
+[win-arm-zip-checksum-main]: https://aka.ms/dotnet/10.0.1xx/daily/dotnet-sdk-win-arm.zip.sha512
 
 [win-arm-badge-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/win_arm_Release_version_badge.svg?no-cache
 [win-arm-version-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/productCommit-win-arm.txt
 [win-arm-zip-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-win-arm.zip
-[win-arm-zip-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-win-arm.zip.sha
+[win-arm-zip-checksum-10.0.1XX-preview2]: https://aka.ms/dotnet/10.0.1xx-preview2/daily/dotnet-sdk-win-arm.zip.sha512
```

Comment:
Same question here for win-arm in .NET 10 (also .NET 9 probably)

Reply:
Same comment as above. The VMR's table doesn't have win-arm anymore.